src/Layers/xrRenderPC_GL/CMakeLists.txt

```
@@ -1,70 +1,436 @@
+    "../xrRender/Light_DB.cpp"
+    "../xrRender/Light_DB.h"
+    "../xrRender/light_gi.cpp"
+    "../xrRender/light_gi.h"
+    "../xrRender/light.h"
+    "../xrRender/Light_Package.cpp"
+    "../xrRender/Light_Package.h"
+    "../xrRender/Light_Render_Direct_ComputeXFS.cpp"
+    "../xrRender/Light_Render_Direct.cpp"
+    "../xrRender/Light_Render_Direct.h"
+    "../xrRender/light_smapvis.cpp"
+    "../xrRender/light_smapvis.h"
+    "../xrRender/LightTrack.cpp"
+    "../xrRender/LightTrack.h"
+    "../xrRender/light_vis.cpp"
+    "../xrRender/ModelPool.cpp"
+    "../xrRender/ModelPool.h"
+    "../xrRender/NvTriStrip.cpp"
+    "../xrRender/NvTriStrip.h"
+    "../xrRender/NvTriStripObjects.cpp"
+    "../xrRender/NvTriStripObjects.h"
+    "../xrRender/occRasterizer_core.cpp"
+    "../xrRender/occRasterizer.cpp"
+    "../xrRender/occRasterizer.h"
+    "../xrRender/ParticleEffect.cpp"
+    "../xrRender/ParticleEffectDef.cpp"
+    "../xrRender/ParticleEffectDef.h"
+    "../xrRender/ParticleEffect.h"
+    "../xrRender/ParticleGroup.cpp"
+    "../xrRender/ParticleGroup.h"
+    "../xrRender/PSLibrary.cpp"
+    "../xrRender/PSLibrary.h"
+    "../xrRender/QueryHelper.h"
+    "../xrRender/R_Backend.cpp"
+    "../xrRender/R_Backend_DBG.cpp"
+    "../xrRender/R_Backend.h"
+    "../xrRender/R_Backend_hemi.cpp"
+    "../xrRender/R_Backend_hemi.h"
+    "../xrRender/R_Backend_Runtime.cpp"
+    "../xrRender/R_Backend_Runtime.h"
+    "../xrRender/R_Backend_tree.cpp"
+    "../xrRender/R_Backend_tree.h"
+    "../xrRender/R_Backend_xform.cpp"
+    "../xrRender/R_Backend_xform.h"
+    "../xrRender/r_constants_cache.h"
+    "../xrRender/r_constants.cpp"
+    "../xrRender/r_constants.h"
+    "../xrRender/r__dsgraph_build.cpp"
+    "../xrRender/r__dsgraph_render.cpp"
+    "../xrRender/r__dsgraph_render_lods.cpp"
+    "../xrRender/r__dsgraph_types.h"
+    "../xrRender/R_DStreams.cpp"
+    "../xrRender/R_DStreams.h"
+    "../xrRender/ResourceManager.cpp"
+    "../xrRender/ResourceManager.h"
+    "../xrRender/ResourceManager_Loader.cpp"
+    "../xrRender/ResourceManager_Reset.cpp"
+    "../xrRender/ResourceManager_Resources.cpp"
+    #"../xrRender/ResourceManager_Scripting.cpp"
+    "../xrRender/r__occlusion.cpp"
+    "../xrRender/r__occlusion.h"
+    "../xrRender/r__pixel_calculator.cpp"
+    "../xrRender/r__pixel_calculator.h"
+    "../xrRender/r__screenshot.cpp"
+    "../xrRender/r__sector.cpp"
+    "../xrRender/r__sector.h"
+    "../xrRender/r__sector_traversal.cpp"
+    "../xrRender/r_sun_cascades.h"
+    "../xrRender/r__sync_point.cpp"
+    "../xrRender/r__sync_point.h"
+    "../xrRender/Shader.cpp"
+    "../xrRender/Shader.h"
+    "../xrRender/ShaderResourceTraits.h"
+    "../xrRender/SH_Atomic.cpp"
+    "../xrRender/SH_Atomic.h"
+    "../xrRender/SH_Constant.cpp"
+    "../xrRender/SH_Constant.h"
+    "../xrRender/SH_Matrix.cpp"
+    "../xrRender/SH_Matrix.h"
+    "../xrRender/SH_RT.h"
+    #"../xrRender/SH_Texture.cpp"
+    #"../xrRender/SH_Texture.h"
```

This exact header should be added.
```suggestion
    "../xrRender/SH_Texture.h"
```